protocol-configuration/mainnetAddresses.ts

```diff
@@ -196,6 +196,26 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0xd10386804959a121a8a487e49f45aa9f5a2eb2a0',
     category: AddressCategory.External
   },
+  ethToDaiLBPSwapper: {
+    artifactName: 'BalancerLBPSwapper',
+    address: '0xf7991f4698ffb6716982aec7f78964dd731c4a54',
+    category: AddressCategory.Core
+  },
+  ethToDaiLensDai: {
+    artifactName: 'BPTLens',
+    address: '0xdf9ff5c077d9f3427ade67ac2d27a864be6f3187',
+    category: AddressCategory.PCV
+  },
+  ethToDaiLensEth: {
+    artifactName: 'BPTLens',
+    address: '0xf24401f6992faeacbc5d6c6991db15b5f8364a1b',
+    category: AddressCategory.PCV
+  },
+  ethToDaiLBPPool: {
+    artifactName: 'IWeightedPool',
+    address: '0x34809aEDF93066b49F638562c42A9751eDb36DF5',
```

Comment:
Would recommend verifying this LBPPool contract and inspecting the variables. Neat trick to verify these LBP pools is to pull the source code of an already verified contract and then use the standard-json input verification method:

```
curl -s [https://api.etherscan.io/api\?module\=contract\&action\=getsourcecode\&address\=0x32F0DC9E2D890BAC95106a65eB82db70Bc58BaDB\&apikey\=AETHERSCANPIKEY](https://api.etherscan.io/api/?module\=contract\&action\=getsourcecode\&address\=0x32F0DC9E2D890BAC95106a65eB82db70Bc58BaDB\&apikey\=AETHERSCANPIKEY) | jq -r '.result[0].SourceCode' | sed 's/{{/{/' | sed 's/}}/}/' > sol.json
```